pkg/catalog/catalog_test.go

```diff
@@ -132,6 +135,16 @@ func TestList(t *testing.T) {
 			wantErr:  true,
 			wantTags: []string{},
 		},
+		{
+			name: "Case 4: Valid image with output tags from current namespace",
+			args: args{
+				name:      "foobar",
+				namespace: "foo",
+				tags:      []string{"1.0.0", "1.0.1", "0.0.1", "latest"},
+			},
+			wantErr:  false,
+			wantTags: []string{"1.0.0", "1.0.1", "0.0.1", "latest"},
+		},
```

Comment:
Check for Actions missing in the existing code. Would be good to add now.

Reply:
Ack